File: POO/J/OzCorp/src/br/com/ozcorp/business/Employee.java
package br.com.ozcorp.business;
import java.text.NumberFormat;
public class Employee {
	private final static NumberFormat format = NumberFormat.getCurrencyInstance();
	
	private String name;
	private String rg;
	private long cpf;
	private String enrollment;
	private String email;
	private String password;
	private Role post;
	private BloodType blood;
	private Gender gender;
	
	protected Employee(String name, String rg, long cpf, String enrollment, String email, String password, Role post,
			BloodType blood, Gender gender) {
		super();
		this.name = name;
		this.rg = rg;
		this.cpf = cpf;
		this.enrollment = enrollment;
		this.email = email;
		this.password = password;
		this.post = post;
		this.blood = blood;
		this.gender = gender;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getRg() {
		return rg;
	}

	public void setRg(String rg) {
		this.rg = rg;
	}

	public long getCpf() {
		return cpf;
	}

	public void setCpf(long cpf) {
		this.cpf = cpf;
	}

	public String getEnrollment() {
		return enrollment;
	}

	public void setEnrollment(String enrollment) {
		this.enrollment = enrollment;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public Role getPost() {
		return post;
	}

	public void setPost(Role post) {
		this.post = post;
	}

	public BloodType getBlood() {
		return blood;
	}

	public void setBlood(BloodType blood) {
		this.blood = blood;
	}

	public Gender getGender() {
		return gender;
	}

	public void setGender(Gender gender) {
		this.gender = gender;
	}
	
	@Override
	public String toString()
    {
        return String.format("Employee's Data:\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n%-17s%s\n",
        		"Name:", name,
        		"RG:", rg,
        		"CPF:", cpf,
        		"Enrollment:", enrollment,
        		"Email:", email,
        		"Password:", password,
        		"Post:", post,
        		"Base Salary:", format.format(post.getBaseSalary()), "Blood:", blood, "Gender:", gender);
    }
}

File: POO/CS/OzCorp/OzCorp/Business/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzCorp.Business
{
    public class Employee
    {
        public String Name { get; set; }
        public String RG { get; set; }
        public long CPF { get; set; }
        public String Enrollment { get; set; }
        public String Email { get; set; }
        public String Password { get; set; }
        public Role Post { get; set; }
        public BloodType Blood { get; set; }
        public Gender Gender { get; set; }
        public Employee(String Name, String RG, long CPF, String Enrollment, String Email, String Password, Role Post, BloodType Blood, Gender Gender)
        {
            this.Name = Name;
            this.RG = RG;
            this.CPF = CPF;
            this.Enrollment = Enrollment;
            this.Email = Email;
            this.Password = Password;
            this.Post = Post;
            this.Blood = Blood;
            this.Gender = Gender;
        }
        public override string ToString()
        {
            return String.Format("Employee's Data:\n{0,-17}{1}\n{2,-17}{3}\n{4,-17}{5}\n{6,-17}{7}\n{8,-17}{9}\n{10,-17}{11}\n{12,-17}{13}\n{14,-17}{15:C2}\n{16,-17}{17}\n{18,-17}{19}\n", "Name:", Name, "RG:", RG, "CPF:", CPF, "Enrollment:", Enrollment, "Email:", Email, "Password:", Password, "Post:", Post, "Base Salary:", Post.BaseSalary, "Blood:", Blood, "Gender:", Gender);
        }
    }
}
